src/backend/api/meals.js

```diff
@@ -2,11 +2,111 @@ const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
+//Returns all meals
 router.get("/", async (request, response) => {
+  let maxPrice = 100000;
+  let title = "%";
+  let createdAfter = 2000;
+  let limit = (await knex("meal")).length;
```

Comment:
Here maybe it would make sense to run a count aggregation instead (both more performant and more clear to the reader what is being done).